### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Nex-opensourcehorray**
hello, brother or sister @linpyaehtet, is it possible for you as a gentleman/lady to make a review on my current solution for Prac6 solutions? I would be grateful if you to take a detailed look and make a specific comment on the coding solution, Thanks!

### CODE REVIEW

File: Prac05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+
+def main():
+    """This function is to execute the matching."""
```
Comment (linpyaehtet) [inline Prac05/state_names.py:13]: Nice docstring :)

File: Prac05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+def get_valid_input():
+    """Deliver the error checking for input."""
+    state_abbreviation = ''
+    is_not_valid_input = True
+    while is_not_valid_input:
+        try:
+            state_abbreviation = input("Enter short state: ").upper()
+            if state_abbreviation == "":
+                break
```
Comment (linpyaehtet) [inline Prac05/state_names.py:34]: You can put **is_not_valid_input = False** instead of break

**Review by linpyaehtet — COMMENTED** [review]
Just minor changes, everything else is good 👍

**Comment by Nex-opensourcehorray** [comment]
@Derrc1kDuaN, hello, bro/sis. Is it convenient for you to take an overall glance at my coding and point out the mistakes I made? I would appreciate your effort. Thanks! (It is supposedly Prac 6 feedback.)

**Pull request merged**